src/main/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataMetricsServiceImpl.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * The MIT License
+ * Copyright © 2017 DTL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package nl.dtls.fairdatapoint.service.impl;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import nl.dtl.fairmetadata4j.model.Metric;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service layer for fair metric
+ *
+ * @author Rajaram Kaliyaperumal <rr.kaliyaperumal@gmail.com>
+ * @author Kees Burger <kees.burger@dtls.nl>
+ * @since 2018-01-16
+ * @version 0.1
+ */
+@Service("fairMetaDataMetricsServiceImpl")
+public class FairMetaDataMetricsServiceImpl {
+    
+    private static final Logger LOGGER = LogManager.getLogger(FairMetaDataServiceImpl.class
+            .getName());
+    
+    private final ValueFactory valueFactory = SimpleValueFactory.getInstance();
+    
+    @Value("${metadataFM.F1.A:nil}")
+    private String mdFmF1A;
+    
+    @Value("${metadataFM.F1.B:nil}")
+    private String mdFmF1B;
+    
+    @Value("${metadataFM.A1.A:nil}")
+    private String mdFmA11;
+    
+    @Value("${metadataFM.A2:nil}")
+    private String mdFmA2;
```

Comment:
These variables names are not necessarily the most readable. Not sure what a better one could be, but we can be creative.